src/output_directory.cpp

```diff
@@ -137,6 +128,17 @@ const char *make_output_directory(const char *exename, const char *jobname) {
   return outdirname;
 }
 
+/* similar to above, but creates a temporary directory */
+const char *make_output_directory() {
+  static char outdirname[] = "meepXXXXXX";
```

Comment:
The name of the output directory should probably include `/tmp` and therefore be changed to `"/tmp/meepXXXXXX"`. Otherwise, the directory is created in the current directory.

Reply:
Oh right.

Getting the tempdir properly is a little annoying.  Here is how [libuv does it](https://github.com/libuv/libuv/blob/7b28d36056d947e74e2c3347ee9bb04a1f5388c6/src/unix/core.c#L1069-L1117).  (libuv is MIT/expat licensed so we can grab its implementation.)

Reply:
Done.

Reply:
Done.